### PR TITLE
feat(unlock-protocol-com) - header issue in staging

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

Fix issue on https://staging.unlock-protocol.com/ where:
 On `Launch App`, the user should be sent to the **staging** app, not to https://app.unlock-protocol.com/locks.

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-protocol-com/scripts/deploy-fleek.sh [hunk 1]
```diff
@@ -6,13 +6,10 @@ set -e
 # This script deploys a static build to fleek.
 
 APP_PATH=$1
-DEPLOY_ENV=$2 #ignored in this script since unlock-protocol.com is always deployed to production
+DEPLOY_ENV=$2
```
Comment (kalidiagne) [inline unlock-protocol-com/scripts/deploy-fleek.sh:9]: do we set the env vars for staging/prod correctly?

Comment (julien51) [reply unlock-protocol-com/scripts/deploy-fleek.sh:9]: We don't actually use fleek anymore... so we can probably clean up that file.

**Review by julien51 — CHANGES_REQUESTED** [review]
We don't use Fleep anymore so I am orettu sure your PR does not do anything :(
Please check CI scripts to see what vars are being set.

**Comment by julien51** [comment]
Please move the fleek changes to _another_ PR. and if you're cleaning this please remove Fleek everywhere

**Pull request closed without merging**